### DISCUSSION TITLE
Why OSDs not expose the service to NodeIP in node as MON and MGR/MDS?

### DISCUSSION THREAD

**Discussion by dannyeuu** [rook/rook#14385] [Q&A]
How to properly use a Ceph cluster in k8s-cluster-01 and mount CephFS on AWS EC2s?

**Comment by travisn** [comment]
See the [ceph networking guide](https://docs.ceph.com/en/latest/rados/configuration/network-config-ref/) for the port ranges used by the OSDs.

**Reply by dannyeuu** [reply]
The problem is that the port is not exposed on pod or node, @travisn:
```
ip-172-31-24-85 kernel: libceph: osd6 (1)172.31.124.239:6801 socket closed (con state V1_BANNER)
```
![Screenshot from 2024-07-02 21-10-05-edited](https://github.com/rook/rook/assets/6808348/892f0b9c-e7cf-4165-84f4-4b01df678959)

**Reply by travisn** [reply]
`172.31.124.239` is the host ip, right? So the issue is just about the ports? Since Ceph uses a range for the OSD ports, it's difficult to declare the ports on the OSD pods. Can you open the OSD port range on your firewall?
